open_spiel/python/mfg/algorithms/mfg_proximal_policy_optimization_test.py

```diff
@@ -0,0 +1,98 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Mean field proximal policy optimaztion."""
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import torch
+import torch.optim as optim
+
+from open_spiel.python import rl_environment
+from open_spiel.python import policy as policy_std
+from open_spiel.python.mfg.algorithms import distribution
+from open_spiel.python.mfg.games import factory
+from open_spiel.python.mfg.algorithms.mfg_proximal_policy_optimization import Agent as mfg_ppo_agent
+from open_spiel.python.mfg.algorithms.mfg_proximal_policy_optimization import learn
+from open_spiel.python.mfg.algorithms.mfg_proximal_policy_optimization import calculate_advantage
+from open_spiel.python.mfg.algorithms.mfg_proximal_policy_optimization import rollout
+from open_spiel.python.mfg.algorithms.mfg_proximal_policy_optimization import calculate_explotability
+from open_spiel.python.mfg.algorithms.mfg_proximal_policy_optimization import Policy as mfg_ppo_policy
+
+
+class PolicyTest(parameterized.TestCase):
+    """Test the policy."""
+    @parameterized.named_parameters(('python', 'mfg_crowd_modelling_2d',
```

Comment:
use consistent delimiters for strings. I suggest double quotes (") as opposed to single quotes (') as it's more consistent with the rest of the code

Reply:
All delimiters were changed to double quotes